javascript/javascript3/week2/script.js

```diff
@@ -0,0 +1,102 @@
+// Movies exercise
+//1-Create an array of bad movies
+fetch(
+  "https://gist.githubusercontent.com/pankaj28843/08f397fcea7c760a99206bcb0ae8d0a4/raw/02d8bc9ec9a73e463b13c44df77a87255def5ab9/movies.json"
+)
+  .then((response) => response.json())
+  .then((movies) => {
+    console.log(movies);
+    const badMovies = movies
+      .filter((movie) => movie.rating <= 4)
+      .map((movie) => movie.title);
+    console.log(badMovies);
+    //2-Creat an array of bad movies since year 2000
+    const twentyOneBadMovies = movies
+      .filter((movie) => movie.year >= 2000)
+      .filter((movie) => movie.rating <= 4)
+      .map((movie) => movie.title);
+    console.log(twentyOneBadMovies);
+  });
+
+//Promise that resolves after set time
+async function waitForfewSeconds(resolveAfter) {
+  await new Promise((resolve) => {
+    setTimeout(() => {
+      resolve();
+    }, resolveAfter * 1000);
+  });
+}
+
+waitForfewSeconds(8).then(() => {
+  console.log("I am called asynchronously");
+});
```

Comment:
It would be nice if you include the 'resolveAfter' parameter inside the console, like `I am called asynchronously after ${resolveAfter} seconds`.

Reply:
I cant its gives me an error as parameter not defined